### ISSUE TITLE
DXF Templates too thick lines

### ISSUE THREAD

**Issue by marcovandersson** [OpenOrienteering/mapper#1224]
### Steps to reproduce
DXF templates are visualized too thick in the dev version, at least according to me. See attached pdf.  When using Karttapullautin 0.3m contour lines from DXF you need thinner lines.

### Actual behaviour
See attachment, too thick lines.

### Expected behaviour
See attachement, thinner lines.
[v0.9 -- 2019-03-26 versus v0.8.4.pdf](https://github.com/OpenOrienteering/mapper/files/3039152/v0.9.--.2019-03-26.versus.v0.8.4.pdf)

### Configuration
Mapper Version: 0.9 2019-03-26 dev
Operating System: Win7, 64 bit.

**Comment by dg0yt** [comment]
In your configuration of Mapper v0.8.4, you probably see a "cosmetic" line (1 pixel width). I know that this is popular for some purposes such as GPX tracks. However, it has problems when pixels are tiny, and this code wasn't maintainable for DXF anyway.

In `dev`, you either get the width, color, and pattern defined in the DXF file, or a default solid purple line of 0.1 mm width. I wouldn't say this is too thick, in absence of any other information. It is less than a regular contour line. I'm curious how others think about it.

In the meantime, you may perhaps just load the DXF as if it was a map, edit the symbol (or even assign more individual symbols, based on the the Layer tag), and save the result as OMAP file for use as a template.

**Comment by Zerbembasqwibo** [comment]
Never used a DXF as template, was not aware of the possibility. So I guess it's OK, if you want other line widths import as map. Of course if you prefer 1 px width, too bad. But as for me, no problems to skip it.

**Comment by dg0yt** [comment]
If you don't want to use the DXF as a template, editing it separately before importing as a map might be even more attractive.

DXF comes from CAD software. Sometimes you get a whole campus map drawing in this format.

**Comment by Zerbembasqwibo** [comment]
I mean I never use a DXF as template (backgruond), I have always put it
through QGIS or imported it to the OOM-format.
Let's hear others' opinions!

Den ons 3 apr. 2019 kl 20:57 skrev Kai Pastor <notifications@github.com>:

**Comment by marcovandersson** [comment]
I would strongly propose to keep the lines as thin as possible (1 or 2
pixels), or make this able to define in settings. I usually uses a lot of
DXF. From Campus, from municipality, from National cadrastral agency,
from Karttapullautin.

Den ons 3 apr. 2019 kl 20:57 skrev Kai Pastor <notifications@github.com>:

**Comment by dg0yt** [comment]
I don't fully understand the relevance of line width when the DXF files are "never used" as templates. Used as a map, you may give arbitrary size and color to the symbols.
 And I also don't know what to do with 25.4 mm : 300 dpi wide lines in printouts.

**Comment by marcovandersson** [comment]
Among all cartographers I know and work with within sweden all of them are
using DXF as templates. As I mentioned, very ofter used when the data
source is the local municipality. So never used is wrong. Instead we should
see this as a very nice feature that is worth to keep. To do what some of
you propose - to import in map, will make the file size huge and difficult
to manage. As template it is easy with F10 to hide all, including DXF
templates. I do not understand the 25.4mm / 300 dpi issue, but I would say
that this is not used for printouts. The use case I describe is related to
cartographic support, as complement to the field observations.

Den mån 8 apr. 2019 kl 07:53 skrev Kai Pastor <notifications@github.com>:

**Comment by dg0yt** [comment]
@marcovandersson All right, but Karttapullautin DXF output, which was refered to in the initial posting, is something entirely different from the AutoCAD DXF output from the local municipality, which comes with style information. Did you try the latter with the GDAL/OGR based importer?

**Comment by mlerjen** [comment]
My workflow so far:
-> Import .dxf into separate mapper project. Filter, define symbols.
-> Export as .png with world file
-> Make .png transparent (photoshop)
-> Use .png as template for the main project

Thus I can choose content and symbolisation of the .dxf myself plus the .png is much smaller than the original .dxf.

**Comment by marcovandersson** [comment]
Yes, but is still the most outstanding template data that one can use then
doing Omaps. So, basically we have 2 very good use cases for reading DXF as
templates, and to visualize with thin lines.

Den mån 8 apr. 2019 22:49Kai Pastor <notifications@github.com> skrev:
